plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -72,7 +73,7 @@ def execute(*)
 
     # check all pods and see if they are running
     # once they are running check if they are stable (for apps only, since jobs are finished and will not change)
-    def wait_for_resources_to_complete(release_docs)
+    def wait_for_resources_to_complete(release_docs, timeout = WAIT_FOR_LIVE)
```

Comment:
Maybe the default value can be removed here, just noticed 😱

Reply:
prefer keyword args and no default value